ocs_ci/ocs/constants.py

```
@@ -97,6 +97,31 @@
+MINIO_INTERNAL_ENDPOINT = "http://minio.minio.svc:9000"
+MINIO_BUCKET = "bucket"
+MINIO_ACCESS_KEY = "minio"
+MINIO_SECRET_KEY = "minio123"
+MOCK_VGRC_NAME = "vgrc-1"
+VOLUME_GROUP_SNAPSHOT_CLASS_CRD_URL = (
+    "https://raw.githubusercontent.com/red-hat-storage/external-snapshotter"
+    "/refs/heads/master/config/crd/bases"
+    "/groupsnapshot.storage.openshift.io_volumegroupsnapshotclasses.yaml"
+)
+LSO_PROVISIONER = "kubernetes.io/no-provisioner"
+VOLSYNC_PSK_SECRET_NAME = "volsync-rsync-tls-secret"
```

Ruff (0.15.17)

[error] Possible hardcoded password assigned to: "VOLSYNC_PSK_SECRET_NAME"

(S105)